networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRootMessageHandler.java

```diff
@@ -84,15 +83,14 @@ public BlobSidecarsByRootMessageHandler(
   @Override
   public Optional<RpcException> validateRequest(
       final String protocolId, final BlobSidecarsByRootRequestMessage request) {
-    // MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK
-    final int maxRequestBlobSidecars = calculateMaxRequestBlobSidecars();
-    if (request.size() > maxRequestBlobSidecars) {
+    final UInt64 maxRequestBlobSidecars = getMaxRequestBlobSidecars();
+    if (maxRequestBlobSidecars.isLessThan(request.size())) {
```

Comment:
I think `request.size.isGreaterThan(maxRequestBlobSidecars)` is more straightforward way of comparison. Do we break a limit? Not a "is limit smaller than we"?